content/en/docs/reference/config/istio.mesh.v1alpha1/index.html

```diff
@@ -2875,14 +2875,15 @@ <h2 id="ProxyConfig">ProxyConfig</h2>
 (<code>sidecar.istio.io/statsInclusionPrefixes</code>,
 <code>sidecar.istio.io/statsInclusionRegexps</code>, and
 <code>sidecar.istio.io/statsInclusionSuffixes</code>). For example, to enable stats
-for circuit breaker, retry, and upstream connections, you can specify stats
+for circuit breaker, retry, upstream connections and request timeout, you can specify stats
```

Comment:
These changes need to be made in the istio/api repo where this file is copied from: https://github.com/istio/api/blob/master/mesh/v1alpha1/istio.mesh.v1alpha1.pb.html#L2876 which is generated from https://github.com/istio/api/blob/master/mesh/v1alpha1/proxy.proto#L561.

Reply:
please see: https://github.com/istio/api/pull/2592